spec/jobs/send_push_notification_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+RSpec.describe Jobs::SendPushNotification do
+  fab!(:user) { Fabricate(:user) }
+  fab!(:subscription) { Fabricate(:push_subscription) }
+  let(:payload) { { notification_type: 1, excerpt: "Hello you" } }
+
+  before do
+    freeze_time
+    SiteSetting.push_notification_time_window_mins = 10
```

Comment:
This is the default value, so not really necessary.

```suggestion
```

Reply:
Since the specs here aren't directly testing the default value, I believe explicitly setting it here prevents the specs from failing in the future if the default value ever changes